Source/WebCore/loader/cache/CachedImage.cpp

```diff
@@ -711,4 +711,11 @@ CachedResource::RevalidationDecision CachedImage::makeRevalidationDecision(Cache
     return CachedResource::makeRevalidationDecision(cachePolicy);
 }
 
+bool CachedImage::mayTryReplaceEncodedData() const {
```

Comment:
This commit is related to Graphics, away from my area of expertise (multimedia). Maybe @magomez might consider to upstream it. Otherwise I'm keeping it downstream only.